mvpa2/tests/test_searchlight_hyperalignment.py

```diff
@@ -208,7 +208,7 @@ def test_searchlight_hyperalignment(self):
         for ds in dss[1:]:
             assert_false(np.all(ds_orig.samples == ds.samples))
         # testing checks
-        slhyp = SearchlightHyperalignment(ref_ds=1, exclude_from_model=[1])
+        slhyp = SearchlightHyperalignment(ref_ds=2, exclude_from_model=[1])
```

Comment:
But shouldn't `ref_ds=1` be as fine/bad as 2 here?
To at least "smoke test" correct behaviour of not considering excluded dataset I think it would be better to check with "boundary" indexes, e.g. 4 (and ideally both with 0 as well).  This way if there is some incorrect indexing happening we would have a better chance to catch it.   Or test it explicit -- if we replace (after copying) .samples of the excluded dataset, results must not change a bit